R/shell.R

```diff
@@ -26,7 +26,13 @@ start_shell <- function(master,
                         packages = NULL,
                         environment = NULL,
                         shell_args = NULL,
-                        app_jar = "sparkr-shell") {
+                        app_jar = NULL) {
+  # by default use sparkapi backend but allow us to pass "sparkr-shell" to test against sparkr
+  if (is.null(app_jar)) {
```

Comment:
I wonder if this is better exposed as a package level option (since it's kind of inside baseball for development mode) rather than an argument to the function.